portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,35 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Arrays;
+import com.google.gson.Gson;
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
Just some convention:

1. The first sentence should say: "A servlet..."
2. End all comments with "." to keep them consistent
3. TODO should come on the next line.
4. Another rule we follow here about TODOs is assigning them to yourself with your ldap, so this would look something like this:

```suggestion
/*
 * Servlet that returns some example content. 
 * TODO(ntarn): Modify this file to handle comments data.
 */
```